libc/strlen.c

```diff
@@ -0,0 +1,10 @@
+size_t strlen(const char* str)
+{
+	size_t len = 0;
+	while (str[len])
```

Comment:
Perhaps change to the following:
```C
size_t strlen(const char* str)
{
    size_t len = 0;
    if(str)
    {
        while (*str++)
        {
            len++;
        }
    }
    
    return len;
}
```
Reason is that I think its a problem with the current implementation if you try to access an element with [] that is out of bounds.